spec/values/calculation/abs.hrx

```diff
@@ -0,0 +1,158 @@
+<===> README.md
```

Comment:
Here (and for each of the other new calculation functions), make sure to test each of the following:

- a CSS variable as an argument
- arguments with units (where appropriate for the function in question)
- something with arithmetic that the browser can handle, but can't be simplified at compile time, like `10px + 10%`
- partial simplification (e.g. `abs(1px + 2px - var(--example))` should simplify to `abs(3px - var(--example))`)

Reply:
This was a nice checklist, I went through all the new files and added tests. Thanks! :)